### ISSUE TITLE
Support check_box in native json

### ISSUE THREAD

**Issue by hngondoki** [OpenSRP/xlson#41]
- [x] The check_box field is XLSForm's select multiple type. This means the spinner variation should have a select_dropdown type

sample xlsform
```
| type                                 | name              | label                                           |
|--------------------------------------|------------------|-------------------------------------------|
| select_multiple problem_hf_pnc      | problem_hf_pnc    | Client condition / problem          |
```

choices

list_name | name | label::English
-- | -- | --
problem_hf_pnc | 1 | Vaginal bleeding
problem_hf_pnc | 2 | Discoloured or watery, liquid vaginal discharge with a bad smell
problem_hf_pnc | 3 | Severe abdominal pain
problem_hf_pnc | 4 | Severe anaemia
problem_hf_pnc | 5 | Convulsions

Expected output format
```JSON
{
    "title": "Danger signs",
    "fields": [
      {
        "key": "danger_signs_present",
        "openmrs_entity_parent": "",
        "openmrs_entity": "concept",
        "openmrs_entity_id": "160939AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
        "type": "check_box",
        "label_info_image_src": "img/chw.png",
        "label_info_has_image": true,
        "label": "Any danger signs present?",
        "label_text_style": "normal",
        "text_color": "#C0C0C0",
        "exclusive": [
          "chk_none"
        ],
        "options": [
          {
            "key": "chk_none",
            "text": "None",
            "value": false,
            "openmrs_entity": "concept",
            "openmrs_entity_id": "1107AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
          },
          {
            "key": "chk_fever",
            "text": "Fever",
            "value": false,
            "openmrs_entity": "concept",
            "openmrs_entity_id": "140238AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
          },
          {
            "key": "chk_prolonged_bleeding",
            "text": "Prolonged bleeding",
            "value": false,
            "openmrs_entity": "concept",
            "openmrs_entity_id": "150802AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
          },
          {
            "key": "chk_severe_headache",
            "text": "Severe headache",
            "value": false,
            "openmrs_entity": "concept",
            "openmrs_entity_id": "139081AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
          },
          {
            "key": "chk_convulsions",
            "text": "Convulsions",
            "value": false,
            "openmrs_entity": "concept",
            "openmrs_entity_id": "164483AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
          },
          {
            "key": "chk_difficulty_breathing",
            "text": "Difficulty breathing",
            "value": false,
            "openmrs_entity": "concept",
            "openmrs_entity_id": "142373AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
          },
          {
            "key": "chk_severe_abdominal_pain",
            "text": "Severe abdominal pain",
            "value": false,
            "openmrs_entity": "concept",
            "openmrs_entity_id": "165271AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
          }
        ],
        "v_required": {
          "value": "true",
          "err": "Please pick at least one"
        }
      }
        ]
}
```

**Comment by allan-on** [comment]
- Added a test for this implementation :heavy_check_mark:

**Comment by WinnyTroy** [comment]
Closed in #43
